### ISSUE TITLE
Bug Report: Week Five Score display needing to change.

### ISSUE THREAD

**Issue by Eliptical** [FunkinCrew/Funkin#575]
This is both a bug report and an enhancement.
Now let me point out the obvious that the floor on Week 5 is snow, which is white.
The color of the text that shows your score is white and very close to the floor.
This problem makes the score text invisible in both Cocoa and Eggnog. It's more visible in Winter Horror land, but barely.
I put this as a bug report as I feel that this is an oversight from the developers rather than something the developers should add.
It also hampers on the game play, sorta. 
Long story short, The text is invisible on Week 5 because it blends in with the floor. Change it to black or something lol
